assets/cubyz/biomes/forest/base.zig.zon

```diff
@@ -1,17 +1,14 @@
 .{
 	.properties = .{
-		.wet,
 	},
 	.minHeight = 22,
 	.maxHeight = 40,
-	
+	.minRadius = 120,
+	.maxRadius = 180,
```

Comment:
I really do not believe that this is enough, but honestly I'm tired arguing with you about the biome sizes.
So I guess we'll just do it your way and see where it gets us.

Reply:
Yeah, that's the problem of the way biomes are samples. You'll have to live with it. I personally don't find it that bad.